src/ConductorSharp.Toolkit/Program.cs

```diff
@@ -4,72 +4,125 @@
+                string textToWrite;
+                if (err.IsVersion())
+                    textToWrite = versionText;
+                else
+                {
+                    textToWrite = HelpText.AutoBuild(
+                        parseResult,
+                        help =>
+                        {
+                            help.Copyright = new CopyrightInfo("Codaxy", 2022);
+                            help.AddPreOptionsLine("Usage: dotnet conductorsharp [options]");
+                            help.Heading = versionText;
+                            return help;
+                        }
+                    );
+                }
 
-                var container = BuildContainer(input);
+                writer.WriteLine(textToWrite);
+            });
+        }
 
-                var commandRegistry = container.Resolve<CommandRegistry>();
+        private static async Task RunToolkit(ToolkitOptions options)
+        {
+            try
+            {
+                if (!File.Exists(options.ConfigurationFilePath))
+                {
+                    Console.Error.WriteLine($"Configuration file {options.ConfigurationFilePath} does not exists");
+                    return;
+                }
 
-                var command = commandRegistry.Get(action);
+                var config = ParseConfigurationFile(options.ConfigurationFilePath);
+                if (!ValidateConfiguration(config))
+                    return;
 
-                await command.Execute(input);
+                var container = BuildContainer(config);
+                var commandRegistry = container.Resolve<CommandRegistry>();
+                // Currently only scaffolding is supported
+                var command = commandRegistry.Get("scaffold");
+                await command.Execute(config);
             }
-            catch (Exception exc)
+            // TODO: Improve error handling
+            catch (Exception ex)
             {
-                PrintHelp();
+                Console.Error.WriteLine($"Exception occured with message: {ex.Message}");
             }
         }
 
-        private static void PrintHelp() => Console.WriteLine("PLACEHOLDER HELP");
+        private static Configuration ParseConfigurationFile(string configFilePath) =>
+            new DeserializerBuilder()
+                .WithNamingConvention(CamelCaseNamingConvention.Instance)
+                .Build()
+                .Deserialize<Configuration>(File.ReadAllText(configFilePath));
 
-        private static IContainer BuildContainer(CommandInput input)
+        private static bool ValidateConfiguration(Configuration config)
+        {
+            if (config.BaseUrl == null)
+                Console.Error.WriteLine("baseUrl property missing in configuration");
+            if (config.Namespace == null)
+                Console.Error.WriteLine("namespace property missing in configuration");
+            if (config.Destination == null)
```

Comment:
We should probably check for null or empty here just in case

Reply:
Agree, fill fix it

Reply:
Done